src/commands/DataCommand.ts

```diff
@@ -0,0 +1,31 @@
+import { config } from "../config";
+import { Command } from "../interactions/interactionClasses";
+import { ChatInputCommandInteraction, SlashCommandBuilder } from "discord.js";
+import { hasAdminRole } from "../util/misc/permissions";
+
+class DataCommand extends Command {
+	constructor() {
+		super("data");
+	}
+
+	async execute(interaction: ChatInputCommandInteraction): Promise<void> {
+		if (await hasAdminRole(interaction)) {
+			await interaction.reply({ files: ["./data/enmap.sqlite"], ephemeral: true });
```

Comment:
Rather export the databases and send json because sending the sqlite file will result in impartial data due to write ahead logging and other such magic.

Reply:
How does this work? Also I do want the sqlite file as well to be able to easily plug it into my current dev version

Reply:
Then also send the -wal and -shm files

Reply:
Done but now that you've mentioned it I think json export would be cool too.